.evergreen/config.yml

```diff
@@ -114,14 +160,31 @@ axes:
       - id: "5.2"
         display_name: "Swift 5.2"
         variables:
-           SWIFT_VERSION: "5.2"
+           SWIFT_VERSION: "5.2.4"
 
 buildvariants:
 
 - matrix_name: "tests-all"
+  display_name: "${swift-version} ${os-fully-featured}"
   matrix_spec:
     os-fully-featured: "*"
     swift-version: "*"
-  display_name: "${swift-version} ${os-fully-featured}}"
   tasks:
     - name: "test"
+
+- matrix_name: "sourcery"
```

Comment:
sourcery has to be run on macOS so this is split up. kind of annoying so we might want to consider doing this on travis or with a github action or something but for now i left it here